osbrain/agent.py

```diff
@@ -252,6 +252,18 @@ def __init__(self, name=None, host=None, serializer=None, transport=None,
     def on_init(self):
         """
         This user-defined method is to be executed after initialization.
+
+        The use of `configure` is preferred over `on_init`, since `configure`
+        is executed when the agent is already running.
```

Comment:
Maybe I would omit this paragraph here. It is not really preferred, it is just different, although it is true that most users would be interested in `.configure()` for most use cases.